### PR TITLE
Template adaptions

### PR DESCRIPTION
**Pull request by FinalAngel**
- [x] HTML Outline
- [x] Validation
- [x] Guidelines

**Overview**
image

**List View**
image

**Detail View**
image

### CODE REVIEW

**Comment by FinalAngel** [comment]
The tests will fail until https://github.com/aldryn/aldryn-faq/pull/60 is merged

File: aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html [hunk 1]
```diff
@@ -3,24 +3,40 @@
...
+            <p class="list-group-item">{% trans "No entry found." %}</p>
+        </div>
     {% endfor %}
 </div>
 
 {% if list and title %}
     <ul class="pager">
-        <li class="previous"><a href="../"><span aria-hidden="true">&larr;</span> {% trans "Back" %}</a></li>
+        <li>
+            <a href="../">
```
Comment (vxsx) [inline aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/includes/question_list.html]: this url feels weird

File: aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/plugins/question_list.html [hunk 2]
```diff
@@ -1,3 +1,5 @@
+{% load i18n %}
```
Comment (vxsx) [inline aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/plugins/question_list.html]: i think we should not load libs that are not used in this particular template.
although it's fairly minor, it still is performance hit

**Comment by intellectronica** [comment]
:+1: (but see @vxsx's comments)

File: aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/plugins/question_list.html (hunk 2 above)
Comment (FinalAngel) [inline aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/plugins/question_list.html]: we had once the discussion that performance is not affected, we use it as good practice to add at least i18n which it is also the case in aldryn-newsblog

**Comment by vxsx** [comment]
LGTM

**Pull request merged**